release/src/main/groovy/run_release_candidate_python_mobile_gaming.sh

```diff
@@ -0,0 +1,167 @@
+#  3. Run UserScore examples with DataflowRunner
+#  4. Run HourlyTeamScore on DirectRunner
+#  5. Run HourlyTeamScore on DataflowRunner
+#
+
+set -e
+set -v
+
+source release/src/main/groovy/python_release_automation_utils.sh
+
+complete() {
+    print_separator "Validation $1"
+    rm -rf $TMPDIR
+}
+
+verify_houly_team_score() {
+    # $1 runner type
+    retry=3
+    should_see='AntiqueBrassPlatypus'
+    while(( $retry >= 0 ))
+    do
+        if [[ $retry > 0 ]]
+        then
+            bq_pull_result=$(bq head -n 100 $DATASET.hourly_team_score_python_$1)
+            if [[ $bq_pull_result = *"$should_see"* ]]
+            then
+                echo "SUCCEED: HourlyTeamScore example successful run on $1-runner"
+                break
+            else
+                retry=$(($retry-1))
+                echo "Did not find team scores, retry left: $retry"
+                sleep 15
+            fi
+        else
+            echo "FAILED: HourlyTeamScore example failed running on $1-runner. Did not found scores of team $should_see in $DATASET.leader_board"
+            complete "FAILED"
+            exit 1
+        fi
+    done
+}
+
+
+print_separator "Start Mobile Gaming Examples"
+echo "SDK version: $VERSION"
+
+TMPDIR=$(mktemp -d)
+echo $TMPDIR
+pushd $TMPDIR
+
+#
+# 1. Download files from RC staging location, install python sdk
+#
+
+wget $CANDIDATE_URL$BEAM_PYTHON_SDK
+print_separator "Creating new virtualenv and installing the SDK"
+virtualenv temp_virtualenv
+. temp_virtualenv/bin/activate
+gcloud_version=$(gcloud --version | head -1 | awk '{print $4}')
+if [[ "$gcloud_version" < "189" ]]
+then
+  update_gcloud
+fi
+pip install $BEAM_PYTHON_SDK[gcp]
+
+
+#
+# 2. Run UserScore with DirectRunner
+#
+
+print_separator "Running userscore example with DirectRunner"
+output_file_name="$USERSCORE_OUTPUT_PREFIX-direct-runner.txt"
+python -m apache_beam.examples.complete.game.user_score \
+--output=$output_file_name \
+--project=$PROJECT_ID \
+--dataset=$DATASET \
+--input=gs://$BUCKET_NAME/5000_gaming_data.csv
+if ls $output_file_name* 1> /dev/null 2>&1;
+then
+	echo "Found output file(s):"
+	ls $output_file_name*
+else
+	echo "ERROR: output file not found."
+	complete "failed when running userscore example with DirectRunner."
+	exit 1
+fi
+echo "SUCCEED: UserScore successfully run on DirectRunner."
+
+
+#
+# 3. Run UserScore with DataflowRunner
+#
+
+print_separator "Running userscore example with DataflowRunner"
+output_file_name="$USERSCORE_OUTPUT_PREFIX-dataflow-runner.txt"
+python -m apache_beam.examples.complete.game.user_score \
+--project=$PROJECT_ID \
+--runner=DataflowRunner \
+--temp_location=gs://$BUCKET_NAME/temp/ \
+--sdk_location=$BEAM_PYTHON_SDK \
+--input=gs://$BUCKET_NAME/5000_gaming_data.csv \
+--output=gs://$BUCKET_NAME/$output_file_name
+# verify results.
+userscore_output_in_gcs="gs://$BUCKET_NAME/$output_file_name"
+gcs_pull_result=$(gsutil ls gs://$BUCKET_NAME)
+if [[ $gcs_pull_result != *$userscore_output_in_gcs* ]]
```

Comment:
Curious why this verification is not similar to `verify_houly_team_score` and is done in a common place. Other that using `ls` and `gsutil ls` they look similar.